src/main/java/Echo.java

```diff
@@ -0,0 +1,175 @@
+                    break;
+                }
+
+                String eventInput = input.substring("event".length()).trim();
+                if (!eventInput.contains("/from ") || !eventInput.contains("/to ")) {
+                    System.out.println("Event time details are missing. Please use '/from' and '/to' to specify the event timing.");
+                    break;
+                }
+
+                String[] eventParts = eventInput.split(" /from ", 2);
+                String[] timeParts = eventParts[1].split(" /to ", 2);
+                if (count == list.length) {
+                    list = Arrays.copyOf(list, count * 2);
+                }
+
+                list[count] = new Event(eventParts[0], timeParts[0], timeParts[1]);
+                count++;
+                System.out.println(break_line);
+                System.out.println("Got it. I've added this task:");
+                System.out.println("  [E][ ] " + eventParts[0] + " (from: " + timeParts[0] + " to: " + timeParts[1] + ")");
+                System.out.println("Now you have " + count + " tasks in the list.");
+                System.out.println(break_line);
+                break;
+
+            case "clearlist":
+                Arrays.fill(list, null); // Clearing the list
+                count = 0; // Resetting the count
+                System.out.println(break_line);
+                System.out.println("The List has been cleared!");
+                System.out.println(break_line);
+                break;
+
+            case "bye":
+                scanner.close();
+                System.out.println("Kkkkkk thanks bye ");
+                return;
+
+            case "mark":
+                if (parts.length > 1) {
+                    int taskNumber = Integer.parseInt(parts[1]) - 1;
+                    if (taskNumber >= 0 && taskNumber < count) {
+                        list[taskNumber].markAsDone();
+                        System.out.println("Yo I have marked task" + (taskNumber + 1) + " as done.");
+                        System.out.println(break_line);
+                        ;
+                    } else {
+                        System.out.println("Invalid task number.");
+                        System.out.println(break_line);
+                    }
+                }
+                break;
+
+            case "unmark":
+                if (parts.length > 1) {
+                    int taskNumber = Integer.parseInt(parts[1]) - 1;
+                    if (taskNumber >= 0 && taskNumber < count) {
+                        list[taskNumber].unmarkAsDone();
+                        System.out.println("Yo I have unmarked task" + (taskNumber + 1));
+                        System.out.println(break_line);
+                    } else {
+                        System.out.println("Invalid task number.");
+                        System.out.println(break_line);
+                    }
+                }
+                break;
+
+            case "help":
```

Comment:
In general, you could consider avoiding some magic literals by refactoring them to variables to adhere to this [guideline](https://nus-cs2113-ay2324s2.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-magic-numbers)